Jenkinsfile_promote_salt_bundle_packages

```diff
@@ -9,7 +9,7 @@ properties([
 pipeline {
 
     parameters {
-        string(defaultValue: '3002.2', description: 'Salt version to promote for Salt Bundle.', name: 'salt_version')
+        string(defaultValue: '3004', description: 'Salt version to promote for Salt Bundle.', name: 'salt_version')
```

Comment:
Do we need this parameter? It's not used at all as I see.

Reply:
Correct. I think we can drop this, as we do not use it in this pipeline.